lib/components/src/empty-state/empty-state.scss

```diff
@@ -0,0 +1,48 @@
+@use '@material/typography/mdc-typography';
+
+.content {
+    width: 632px; 
```

Comment:
i dont think we should be setting the width and the height here. can we remove this and if its needed code in story example